clients/drcachesim/tools/invariant_checker.h

```diff
@@ -143,6 +147,12 @@ class invariant_checker_t : public analysis_tool_t {
             uint num_memory_read_access = 0;
             uint num_memory_write_access = 0;
             addr_t branch_target = 0;
+            bool is_prefetch = false;
+            int opcode = 0;
```

Comment:
Hmm, similar comment to before: having the opcode and then also is_syscall, is_xsave, etc. seems redundant.

Reply:
See other comment. It seems better to re-use the instr_is_x routines where available.